web/app.fluidity.money/app/routes/$network/dashboard/airdrop/common.tsx

```diff
@@ -521,7 +522,7 @@ interface IStakingNowModal {
     slippage: BN,
     maxTimestamp: BN
   ) => Promise<StakingDepositsRes | undefined>;
-  ratios: StakingRatioRes | null;
+  getRatios?: () => Promise<StakingRatioRes | undefined>;
```

Comment:
Have a comment explaining why this oftentimes returns undefined if there was an underlying error, and later control the promise to have it not return undefined at all